packages/react-core/src/helpers/util.ts

```diff
@@ -39,24 +39,25 @@ export function debounce(this: any, func: (...args: any[]) => any, wait: number)
  * @param {HTMLElement} container  The container to check if the element is in view of.
  * @param {HTMLElement} element    The element to check if it is view
  * @param {boolean} partial   true if partial view is allowed
+ * @param {boolean} strict    true if strict mode is set, never consider the container width and element width
  *
  * @returns { boolean } True if the component is in View.
  */
-export function isElementInView(container: HTMLElement, element: HTMLElement, partial: boolean) {
+export function isElementInView(container: HTMLElement, element: HTMLElement, partial: boolean, strict?: boolean) {
```

Comment:
I'm thinking it might be better to default this value to false, so `strict: boolean = false` so we don't have to count on undefined being checked correctly down below.

Reply:
Will update, thanks!